frontend/appflowy_flutter/lib/plugins/database/application/field/field_editor_bloc.dart

```diff
@@ -175,7 +184,7 @@ class FieldEditorEvent with _$FieldEditorEvent {
 @freezed
 class FieldEditorState with _$FieldEditorState {
   const factory FieldEditorState({
-    required final FieldInfo field,
+    required final FieldInfo? field,
```

Comment:
```suggestion
    @Default(null) FieldInfo? field,
```

Reply:
Decided to not feed null into constructor, should I still do this regardless?

Reply:
It's nullable, so we might as well default it I guess.

Reply:
Sure thing